replication/binlogsyncer.go

```diff
@@ -514,10 +514,7 @@ func (b *BinlogSyncer) replySemiSyncACK(p Position) error {
 		return errors.Trace(err)
 	}
 
-	_, err = b.c.ReadOKPacket()
-	if err != nil {
-	}
-	return errors.Trace(err)
+	return nil
```

Comment:
I remember that I referred old manual and wrote the code here.

where can we know the master will not reply?